MBBSEmu/HostProcess/ExportedModules/Majorbbs.cs

```diff
@@ -3009,10 +3012,27 @@ private void sameto()
             Registers.AX = (ushort)(string2Buffer.Contains(string1Buffer, StringComparison.CurrentCultureIgnoreCase) ? 1 : 0);
         }
 
+        /// <summary>
+        ///     Compares ending of strings, checking if the first string ends with the second string (ignoring case)
+        ///
+        ///     Signature: int samend(char *longs, char *ends)
+        ///     Returns: AX == 1, match
+        /// </summary>
+        private void samend()
+        {
+            var string1Pointer = GetParameterPointer(0);
```

Comment:
You can use `GetParameterString` here and avoid the buffer declarations below

Reply:
Nice, fixed.